examples/flash-sale/pages/collections/[handle].js

```diff
@@ -0,0 +1,233 @@
+import { useState } from 'react';
+import { useRouter } from 'next/router';
+import { nacelleClient } from 'services';
+import ProductCard from 'components/ProductCard';
+import styles from 'styles/Collection.module.css';
+import Countdown, { formatTimeDelta, zeroPad } from 'react-countdown';
+
+function Collection(props) {
+  const router = useRouter();
+  const { collection, flashSale } = props;
+  const [products, setProducts] = useState(props.products);
+  const [canFetch, setCanFetch] = useState(props.canFetch);
+  const [isFetching, setIsFetching] = useState(false);
+
+  const activeProducts = canFetch
+    ? products?.slice(0, products.length - 1)
+    : products;
+
+  let flashSaleText = false;
+  const flashSaleRenderer = (timeDelta) => {
+    const formattedDate = formatTimeDelta(timeDelta, {
+      daysInHours: true
+    });
+    const { hours, minutes, seconds, completed } = formattedDate;
+    if (completed) {
+      return <></>;
+    } else {
+      return (
+        <p suppressHydrationWarning>
```

Comment:
More of a question than a request.  Is `suppressHydrationWarning` a good approach?  How does it stack up to conditional rendering like `typeof window !== 'undefined' { // client side code }`

Reply:
I added it to get rid of a console error: https://github.com/ndresx/react-countdown#why-do-i-get-this-error-warning-text-content-did-not-match